srvutil/src/main/java/org/apache/rocketmq/srvutil/ShutdownHookThread.java

```diff
@@ -23,22 +23,22 @@
 import java.util.concurrent.atomic.AtomicInteger;
 
 /**
- * {@link ShutdownHookThread} is the standard hook for filtersrv and namesrv modules.
+ * {@link ShutdownHookThread} is the standard hook for controller and namesrv modules.
  * Through {@link Callable} interface, this hook can customization operations in anywhere.
  */
 public class ShutdownHookThread extends Thread {
     private volatile boolean hasShutdown = false;
-    private AtomicInteger shutdownTimes = new AtomicInteger(0);
+    private final AtomicInteger shutdownTimes = new AtomicInteger(0);
     private final Logger log;
-    private final Callable callback;
+    private final Callable<Void> callback;
```

Comment:
Is it `Callable` still necessary?

Reply:
Should change it to `Runnbale` since no return value is needed.

Reply:
> Is it `Callable` still necessary?

Should change it to `Runnable` since no return value is needed.

Reply:
@caigy done, PTAL.